src/devices/ServoMotor/ServoMotor.sln

```diff
@@ -0,0 +1,70 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{A6C9C63E-8347-4956-8816-C5D463194BCA}"
```

Comment:
This is also weird

Reply:
See my comment above, but in general it is very common for developers who are coding a library to also have their unit test project part of the same solution, that is what the intention is here. Most developers that add a unit test project to their binding, will want to have it loaded in VS in order to be able to quickly run and debug the tests right from there.